applications/nrf_desktop/doc/dvfs.rst

```diff
@@ -0,0 +1,104 @@
+.. _nrf_desktop_dvfs:
+
+DVFS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Desktop Dynamic Voltage and Frequency Scaling (DVFS) module is responsible for switching the frequency and voltage according to the application needs.
```

Comment:
Will the Arch diagram also require an update? - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/nrf_desktop/description.html#module_and_component_overview

Reply:
We have a ticket for that: https://nordicsemi.atlassian.net/browse/NCSDK-28811